src/components/breadcrumb/_breadcrumb.scss

```diff
@@ -31,28 +31,34 @@ $breadcrumb-chevron-height: 0.65rem;
         content: none;
       }
 
-      &:not(:nth-last-child(2)) {
+      &:not(:nth-last-child(1)) {
         display: inline-block;
       }
 
+      &:first-child:nth-last-child(1) {
+        &:before {
+          @include icon('chevron-left', $breadcrumb-chevron-height, $breadcrumb-chevron-height);
+          content: '';
+          margin: 0 0.2rem 0 0;
+          vertical-align: middle;
+        }
+      }
+
       &:not(:last-child):after {
         @include icon('chevron-right', $breadcrumb-chevron-height, $breadcrumb-chevron-height);
         content: '';
-        vertical-align: middle;
         margin: 0;
+        width: 1.25rem;
 
         // We have to override the icon settings so it renders correctly in ie11
         background-position: center center;
-        width: 1.25rem;
+        vertical-align: middle;
       }
     }
-
-    &--current {
-      color: $color-text-light;
-    }
   }
 
   &__link {
-    text-decoration: none;
+    color: #222;
+    text-decoration: underline;
```

Comment:
Removing the underline on hover is needed for a11y (shouldn't rely on colour change alone) but we could leave for a full sweep of this in: https://trello.com/c/mA0eWaT8/545-issue-803-dont-use-colour-as-the-only-indicator-of-hover-states